lib/ansible/module_utils/network/meraki/meraki.py

```diff
@@ -222,6 +223,9 @@ def get_net_id(self, org_name=None, net_name=None, data=None):
         self.fail_json(msg='No network found with the name {0}'.format(net_name))
 
     def construct_path(self, action, function=None, org_id=None, net_id=None, org_name=None):
+        """ Build a path from the URL catalog
+            Intelligently inserts org_id and net_id
```

Comment:
Please have an empty line between the first line and the long description. You'll also want to dedent the paragraph part.